docs/FRSKY_R9.md

```diff
@@ -54,24 +54,30 @@ If you have the older 2018, "ACCST" version of the R9M, you will want to perform
 
 To install the sketch on the M5Stamp C3U Mate, use the Arduino IDE. Open the mlrs-wireless-bridge.ino sketch from the mLRS esp/mlrs-wireless-bridge folder, edit the mlrs-wireless-bridge.ino file to uncomment only the MODULE\_M5STAMP\_C3U\_MATE\_FOR\_FRSKY\_R9M define, select the ESP32C3 Dev board in the IDE, connect the M5Stamp C3U Mate module USB connector to your computer while holding down the center button, and upload the sketch via the IDE.
 
 __Be sure to unplug the M5Stamp C3U Mate from the back of the R9M when programming via USB to avoid feeding 5 volt power back to R9M which might cause damage.__
 
 <img src="images/Frsky_R9M_M5Stamp_C3U_installed.jpg" width="360px">
 
-Connecting the M5Stamp C3U Mate to the R9M is easy: Remove the screw and the plastic cover. Cut a 5 pin length of the included pin header female connector and pull out the pin from the last position. This position serves as a key to avoid plugging in the board incorrectly. Solder the pins in the thru holes as shown below (key position hanging over the left) and reinstall the plastic cover.  After programming the sketch, install on the back of the R9M as shown above. Then, you can connect your GCS computer or mobile device to the "mLRS AP UDP" WiFi access point and connect the GCS via UDP on port 14550.
+Connecting the M5Stamp C3U or Pico Mate to the R9M is easy: Remove the screw and the plastic cover. Cut a 5 pin length of the included pin header female connector and pull out the pin from the last position. This position serves as a key to avoid plugging in the board incorrectly. Solder the pins in the thru holes as shown below (key position hanging over the left) and reinstall the plastic cover.  After programming the sketch, install on the back of the R9M as shown above. Then, you can connect your GCS computer or mobile device to the "mLRS AP UDP" WiFi access point and connect the GCS via UDP on port 14550.
 
 <img src="images/Frsky_R9M_M5Stamp_C3U_header.jpg" width="360px">
 
+If you use the M5Stamp Pico Mate, you can select the Bluetooth protocol when you edit the mlrs-wireless-bridge.ino file and then you can connect your GCS via Bluetooth SPP.  The Bluetooth option can be very convenient, especially in cases where WiFi might be used for an Internet connection on your GCS computer or device.
```

Comment:
For this line: 'GCS via Bluetooth SPP' think I would prefer 'GCS via Bluetooth' or 'GCS via Bluetooth Classic' to try and stay consistent with other mentions.

Reply:
OK, but I'd like to mention the official name for this, "Serial Port Profile" or "SPP" somewhere as it makes it easier for a reader to further research the issues around the older standards which supported SPP and the newer BLE standards which don't.  Would it be better to move the SPP reference up to where we first mention "Bluetooth Classic" mode?

Reply:
I have the same comment. I would suggest to call it Bluetooth Classic everywhere, It has become super common to call it so. 
Also, in my understanding, it's not really correct that the ESP woudl support SSP, since SSP is a driver thing while BL Classic is the hardware thing. In none of the docs for any of the ESP modules you would find whem mentioning SSP (unless there is a section on how to connect with BT).  
I understand your intention but I find it just super confusing. If we want to help with telling how to connect a PC with the ESP32, I think it should go in a separate section somewhere.

Reply:
I don't see why it matters to a user which layer is supported by hardware vs drivers and I don't think I ever said the hardware directly supports SPP.  It's the newer BLE standards which dropped support for SPP and SPP is what is needed to talk to existing GCS via Bluetooth.  I think mentioning the official name of the profile is more likely to help someone wanting to understand why the new chips won't connect with their GCS via BT than just the imprecise term "classic".  But I don't want to waste time arguing about something so unimportant so I'll remove the reference to SPP.